crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/rpc.rs

```diff
@@ -46,35 +46,40 @@ impl UsedResources {
     }
 }
 
-/// Represents contract output, along with the data and the resources consumed during execution
+/// Represents call output, along with the data and the resources consumed during execution
 #[derive(Debug)]
-pub struct CallContractOutput {
-    pub result: CallContractResult,
+pub struct CallOutput {
+    pub result: CallResult,
     pub used_resources: UsedResources,
 }
 
-/// Enum representing possible contract execution result, along with the data
+/// Enum representing possible call execution result, along with the data
 #[derive(Debug)]
-pub enum CallContractResult {
+pub enum CallResult {
     Success { ret_data: Vec<Felt252> },
-    Failure(CallContractFailure),
+    Failure(CallFailure),
 }
 
 /// Enum representing possible call failure and its' type.
 /// `Panic` - Recoverable, meant to be caught by the user.
 /// `Error` - Unrecoverable, equivalent of panic! in rust.
 #[derive(Debug)]
-pub enum CallContractFailure {
+pub enum CallFailure {
     Panic { panic_data: Vec<Felt252> },
     Error { msg: String },
 }
 
-impl CallContractFailure {
+pub enum AddressOrClassHash {
```

Comment:
Struct introduced to make sure we can handle both the library call and contract call with the same functions (used in `CallFailure::from_execution_error`)